sklearn/linear_model/logistic.py

```diff
@@ -500,6 +500,7 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
         number for verbosity.
 
     solver : {'lbfgs', 'newton-cg', 'liblinear', 'sag', 'saga'}
+        default: 'default'. Will be changed to 'auto' solver in 0.22.
```

Comment:
default 'default' doesn't mean anything. If you're going to say such a thing, you need to explain what it means. Better to say it is lbfgs by default, even if the string 'default' is used as a placeholder. 'auto' needs to be added to the list of options and described.